moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -1040,32 +1040,41 @@ bool IKFastKinematicsPlugin::getPositionIK(const geometry_msgs::Pose& ik_pose, c
 
   IkSolutionList<IkReal> solutions;
   int numsol = solve(frame, vfree, solutions);
-  std::vector<std::vector<double>> solutions_obey_limits;
-  std::vector<double> dists;
-  bool solution_found = false;
-
   ROS_DEBUG_STREAM_NAMED("ikfast", "Found " << numsol << " solutions from IKFast");
 
+  // struct for storing and sorting solutions
+  struct limit_obeying_sol
```

Comment:
This struct name may not be following  the naming [convention](https://google.github.io/styleguide/cppguide.html#Type_Names)